docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo
+    container_name: mongodb
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
```

Comment:
Please add the environment variables and a quick Docker instruction in the README